sse/src/main/kotlin/io/ktor/samples/sse/SseApplication.kt

```diff
@@ -29,15 +31,15 @@ fun main() {
 data class SseEvent(val data: String, val event: String? = null, val id: String? = null)
 
 /**
- * Method that responds an [ApplicationCall] by reading all the [SseEvent]s from the specified [events] [ReceiveChannel]
+ * Method that responds an [ApplicationCall] by reading all the [SseEvent]s from the specified [eventFlow] [Flow]
  * and serializing them in a way that is compatible with the Server-Sent Events specification.
  *
  * You can read more about it here: https://www.html5rocks.com/en/tutorials/eventsource/basics/
  */
-suspend fun ApplicationCall.respondSse(events: ReceiveChannel<SseEvent>) {
+suspend fun ApplicationCall.respondSse(eventFlow: Flow<SseEvent>) {
     response.cacheControl(CacheControl.NoCache(null))
     respondTextWriter(contentType = ContentType.Text.EventStream) {
```

Comment:
can you please update this to use `respondBytesWriter` instead? it is better because it avoids blocking java apis.

Reply:
thanks. merged!